base/applications/mspaint/lang/ro-RO.rc

```diff
@@ -105,18 +105,18 @@ ID_POPUPMENU MENU
 BEGIN
     POPUP "Popup #0"
     BEGIN
-        MENUITEM "C&ut\tCtrl+X", IDM_EDITCUT
-        MENUITEM "Copy\tCtrl+C", IDM_EDITCOPY
-        MENUITEM "&Paste\tCtrl+V", IDM_EDITPASTE
-        MENUITEM "De&lete selection\tDel", IDM_EDITDELETESELECTION
-        MENUITEM "Select &All\tCtrl+A", IDM_EDITSELECTALL
+        MENUITEM "&Decupează\tCtrl+X", IDM_EDITCUT
+        MENUITEM "Copiază\tCtrl+C", IDM_EDITCOPY
+        MENUITEM "&Lipește\tCtrl+V", IDM_EDITPASTE
+        MENUITEM "Ște&rge selecția\tDel", IDM_EDITDELETESELECTION
+        MENUITEM "Selectea&ză &tot\tCtrl+A", IDM_EDITSELECTALL
```

Comment:
This has two keys marked as actionable - "Z" and "T". Assign only one.
Also, "T" (especially the lower case one) is not a good candidate for action key, for it gets painted too similar to "Ț" (T with comma).